doc/api/fs.markdown

```diff
@@ -822,13 +822,20 @@ Returns a new WriteStream object (See `Writable Stream`).
     { flags: 'w',
       encoding: null,
       fd: null,
-      mode: 0666 }
+      mode: 0666,
+      autoClose: true }
 
 `options` may also include a `start` option to allow writing data at
 some position past the beginning of the file.  Modifying a file rather
 than replacing it may require a `flags` mode of `r+` rather than the
 default mode `w`.
 
+If `autoClose` is false, then the file descriptor won't be closed, even if
+there's an error. It is your responsiblity to close it and make sure
```

Comment:
I can change this line but i copied this text from createReadStream autoclose paragraph. Shall i still change it? As of now both createReadStream and createWriteStream para are consistent.